core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSearchRequestBuilder.java

```diff
@@ -98,6 +98,17 @@ public ConceptSearchRequestBuilder filterByExclusions(Iterable<String> exclusion
 	public ConceptSearchRequestBuilder filterByTermType(String termType) {
 		return addOption(OptionKey.TERM_TYPE, termType);
 	}
+
+	/**
+	 * Filters concepts by their type.
+	 * 
+	 * @param type
+	 *            - String representation of the concept type filtering
+	 * @return
+	 */
+	public ConceptSearchRequestBuilder filterByType(String type) {
```

Comment:
Add multi-valued `filterByType` method and also two more with the `CodeType` and `Iterable<CodeType>` as parameter.